src/main/java/frc/robot/commands/DropBothPlacers.java

```diff
@@ -0,0 +1,16 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import frc.robot.subsystems.ClimberPlacer;
+
+public class DropBothPlacers extends ParallelCommandGroup {
```

Comment:
You are using this very simple `ParallelCommandGroup` only once, does it really deserve a class?

Reply:
Apparently yes. Ask Riki about single responsibility or some shit like that.